mobify/progressive-web-scaffold
# Notifications

Port of the notification system from [Crabtree](https://github.com/mobify/crabtree-evelyn-progressive) ([original PR here](https://github.com/mobify/crabtree-evelyn-progressive/pull/52))

Renamed to Notifications. Required for Progressive Checkout. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-808

## Changes
- Add Notification system

## How to test-drive this PR
- In `app/containers/pdp/actions.js` add the following import:
`import {addNotification} from '../app/actions'`
then within `submitCartForm` add the following (line 23):
`        dispatch(addNotification({
            content: 'Hello message',
            id: 'charitableSubmissionMessage',
            showRemoveButton: true
        }))
`
- Preview and add an item on a PDP
- Observe the notification pops into view
- Observe that tapping the 'x' button in the notification dismisses it.
